.travis.yml

```diff
@@ -70,6 +70,11 @@ script:
     - export LD_LIBRARY_PATH=$TRAVIS_BUILD_DIR/depends/$HOST/lib
     - if [ "$RUN_TESTS" = "true" ]; then make $MAKEJOBS check VERBOSE=1; fi
     - if [ "$RUN_TESTS" = "true" ]; then qa/pull-tester/rpc-tests.py --coverage; fi
+    - if [ "$RUN_FEDPEG_BITCOIND" = "true" ]; then BITCOIND_VERSION=0.16.3; fi
```

Comment:
any particular reason this can't be one big block inside one conditional?

Reply:
Just readability I guess. You prefer it alltogether? I know it's a bit stupid to take 5 lines for one extra thing, but those files are already so unreadable, didn't want to contribute more to that.

I could add all the install stuff in one line and then the actual test in a second. Will be one long line though :D

Reply:
Can't set the variables in the same line as doing 3 commands they don't ripple through. So now have var line, install line and test line.